vmware/docker-volume-vsphere
# E2E tests related to default vmgroup and non-default vmgroup

Fixes: #1486 and #1477 

This PR fixes following issues:
- 1. TestDeleteDefaultVmgroup - #1486
- 2. Fixing test cleanup for unit test vmdk_ops_test.py - Issue #1477

 TestDeleteDefaultVmgroup has already been reviewed.

Fixing test cleanup is the new change in the current commit. Please review it.

=============================

**Test: TestDeleteDefaultVmgroup**

```
Steps:
Objective: Verify volume creation fails after deleting _DEFAULT tenant
 1. Create a volume
 2. Verify volume from docker host and esx
 2. Delete default tenant
 3. Verify not able to create volume from docker host - Error: VM does not belong to any vmgroup
 4. Verify docker volume ls does not show any volumes
 5. Verify admin cli shows volume with VMGroup set as N/A
 6. Create _DEFAULT vmgroup with —default-datastore=_VM_DS
 7. Again create new volume - operation should succeed
 8. Verify volume from esx and docker host
```




```
2017/06/13 22:20:49 Removing policy [validPolicy] on esx [10.162.4.8]
--- PASS: Test (463.98s)
PASS
OK: 25 passed
ok  	github.com/vmware/docker-volume-vsphere/tests/e2e	463.982s
make: Leaving directory `/go/src/github.com/vmware/docker-volume-vsphere/vmdk_plugin'

```
